client.js

```diff
@@ -111,9 +113,15 @@ DHT.prototype._setBucketCheckInterval = function () {
           // retry with initial bootstrap nodes
           self._bootstrap(true)
         }
+        
+        queueNext()
```

Comment:
if it is in the if, the interval will stall as soon as the bucket is not outdated (15min)

`if (diff >= self._bucketOutdatedTimeSpan) {`

Reply:
ah right, nice catch